code/game/machinery/doors/airlock_types.dm

```diff
@@ -54,6 +54,10 @@
 	icon = 'icons/obj/doors/airlocks/station/science.dmi'
 	assemblytype = /obj/structure/door_assembly/door_assembly_science
 
+/obj/machinery/door/airlock/virology
```

Comment:
You need to also add a `/obj/machinery/door/airlock/virology/glass` for the code to compile, did you test this before PRing?

Reply:
oops, I did test it a few days ago but that was just replacing the sprites, i'll fix it now